dora/tests/integration/src/test/java/alluxio/client/fs/io/FileOutStreamIntegrationTest.java

```diff
@@ -161,7 +169,7 @@ public void writeTwoByteArrays() throws Exception {
    */
   @Test
   public void longWrite() throws Exception {
-    AlluxioURI filePath = new AlluxioURI(PathUtils.uniqPath());
+    AlluxioURI filePath = new AlluxioURI(mPath.toString());
```

Comment:
are you reusing the same file for every test? In `@After` I dont think the file is deleted between test cases so this kind of overwrite is really shady. Can you use a different path for every test case?

Reply:
`mPath` is a child path of a `TemporaryFolder`, which as I understanded will be deleted when the test method end.